File: Python/Garage.py
#import library
from ParkingLot import ParkingLot

#deklarasi class
class Garage:
    #private atribut
    __namaGarasi = ""
    __luasGarasi = ""
    __daftarKendaraan = ""
    __parking = []
    

    #constructor
    def __init__(self, namaGarasi = "", luasGarasi = "", daftarKendaraan = ""):
        self.__namaGarasi = namaGarasi
        self.__luasGarasi = luasGarasi
        self.__daftarKendaraan = daftarKendaraan
        self.__parking = []
    
    #setter
    def setnamaGarasi(self, namaGarasi):
        self.__namaGarasi = namaGarasi
    def setluasGarasi(self, luasGarasi):
        self.__luasGarasi = luasGarasi
    def setdaftarKendaraan(self, daftarKendaraan):
        self.__daftarKendaraan = daftarKendaraan
    def setparking(self, parking):
        self.__parking.append(parking)
    
    #getter
    def getnamaGarasi(self):
        return self.__namaGarasi
    def getluasGarasi(self):
        return self.__luasGarasi
    def getdaftarKendaraan(self):
        return self.__daftarKendaraan
    def getparking(self):
        return self.__parking

File: C++/Garage.cpp
#pragma once
// library
#include <iostream>
#include <string.h>

using namespace std;

class Garage
{
    // atribut private
private:
    string nama;
    string luas;
    string daftar;

public:
    // constructor
    Garage()
    {
    }
    Garage(string nama, string luas, string daftar)
    {
        this->nama = nama;
        this->luas = luas;
        this->daftar = daftar;
    }

    // setter
    void setNama(string nama)
    {
        this->nama = nama;
    }
    void setLuas(string luas)
    {
        this->luas = luas;
    }
    void setDaftar(string daftar)
    {
        this->daftar = daftar;
    }

    // getter
    string getNama()
    {
        return this->nama;
    }
    string getLuas()
    {
        return this->luas;
    }
    string getDaftar()
    {
        return this->daftar;
    }

    // destructor
    ~Garage()
    {
    }
};